index.js

```diff
@@ -21,6 +21,15 @@ function getPathWithQueryStringParams(event) {
   return url.format({ pathname: event.path, query: event.queryStringParameters })
 }
 
+function getContentType(params) {
+  // only compare mime type; ignore encoding part
+  return params.contentTypeHeader ? params.contentTypeHeader.split(';')[0] : ''
+}
+
+function isContentTypeBinaryMimeType(params) {
```

Comment:
Do you mean major version bump? I considered this, but this is a bug fix that we want applied to version 2. However, there is a possibility to break anyone who fixed this on their Express side (e.g. via middleware).

Regardless of which we choose, the version bump will be a separate commit.